code/game/objects/items/tools/flame_tools.dm

```diff
@@ -370,6 +371,58 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 	STOP_PROCESSING(SSobj, src)
 	qdel(src)
 
+/obj/item/clothing/mask/cigarette/antitox
+	name = "Neurokiller cig"
+	desc = "a new type of cig developed recently to fend off toxic gasses, might still tire you."
+	icon_state = "anticigoff"
+	item_state = "anticigoff"
+	icon_on = "chemcigon"
+	smoketime = 30
+	chem_volume = 60
+	transquantity = 2 // one of each for the whole duration
+	list_reagents = list(/datum/reagent/medicine/ryetalyn = 30, /datum/reagent/water = 30)  //some water so it purges the rye too
+
+/obj/item/clothing/mask/cigarette/emergency
+	name = "красный русский"
+	desc = "its got a big warning about clone damage, use as a last resort."
+	icon_state = "rrcigoff"
+	item_state = "rrcigoff"
+	icon_on = "chemcigon"
+	smoketime = 10
+	transquantity = 1
+	list_reagents = list(/datum/reagent/medicine/russian_red = 10)  //same ammount as a pill
+
+/obj/item/clothing/mask/cigarette/bica
+	name = "strawberry flavored cigarette"
+	desc = "its red tipped. and got bold letters that read(BICA)"
+	icon_state = "bicacigoff"
+	item_state = "bicacigoff"
+	icon_on = "chemcigon"
+	smoketime = 30
+	transquantity = 5 // one of each for the whole duration
+	list_reagents = list(/datum/reagent/medicine/bicaridine = 15)
+
+/obj/item/clothing/mask/cigarette/kelo
+	name = "lemon flavored cigarette"
+	desc = "its yellow tipped. and got bold letters that read(KELO)."
+	icon_state = "kelocigoff"
+	item_state = "kelocigoff"
+	icon_on = "chemcigon"
+	smoketime = 30
+	transquantity = 5 // one of each for the whole duration
+	list_reagents = list(/datum/reagent/medicine/kelotane = 15)
+
+/obj/item/clothing/mask/cigarette/tram
+	name = "poppy flavored cigarette"
+	desc = "terragov opioid alternative, recent production is a bit dilluted, but still effective."
+	icon_state = "tramcigoff"
+	item_state = "tramcigoff"
+	icon_on = "chemcigon"
+	smoketime = 60
+	chem_volume = 120
+	transquantity = 2 // one of each for the whole duration
+	list_reagents = list(/datum/reagent/medicine/tramadol = 60, /datum/reagent/water = 60)
```

Comment:
60 seems a lot, no?

Reply:
its a minute. i mean. people can use valk and just pop pills. this will only keep 1u of trama and purge other shit

Reply:
make it way less
also smoketime is in 2 second intervals not 1 second